### PR TITLE
[WIP] Create serverless traffic splitting form

### PR DESCRIPTION
**Pull request by nemesis09**
Intends to address issue-

- https://jira.coreos.com/browse/ODC-850

This PR-
-Creates a form for serverless traffic splitting

Screens-
Peek 2019-10-03 03-42

### CODE REVIEW

File: frontend/packages/dev-console/src/components/import/ImportForm.tsx [hunk 1]
```diff
@@ -122,6 +122,9 @@ const ImportForm: React.FC<ImportFormProps & StateProps> = ({
         limitUnit: 'Mi',
       },
     },
+    trafficSplit: {
+      value: 'latest'
```
Comment (invincibleJai) [inline frontend/packages/dev-console/src/components/import/ImportForm.tsx]: will there be just one `value`?

**Comment by debsmita1** [comment]
@nemesis09  Please update the GIF when your PR is ready and please do take care of the lint errors.

**Comment by rohitkrai03** [comment]
@nemesis09 You have 22 lint errors in your code. Please run linter before you push any code or setup `eslint` and `prettier` with VS Code which works very well.

**Comment by sahil143** [comment]
@nemesis09 Please close this Pr as I raised another Pr: https://github.com/openshift/console/pull/3182

**Pull request closed without merging**